File: 171. Excel Sheet Column Number.cpp
class Solution {
public:
    int titleToNumber(string s) {
        int base = 1;
        int res = 0;
        for (int i = s.size()-1; i >= 0; i--)
        {
            res += (s[i] - 'A' +1) *base;
            base *= 26;
        }
        return res;
    }
};

File: 171. Excel Sheet Column Number.py
class Solution(object):
    def titleToNumber(self, s):
        """
        :type s: str
        :rtype: int
        """
        base = 1
        res = 0
        for i in xrange(len(s)-1, -1, -1):
            res += (ord(s[i]) - ord('A') + 1) * base
            base *= 26
        return res
